run_pacta_data_preparation.R

```diff
@@ -103,6 +103,16 @@ logger::log_info(
 
 scenario_raw_data_to_include <- lapply(scenario_raw_data_to_include, get, envir = asNamespace("pacta.scenario.preparation"))
 
+factset_timestamp <-
+  unique(sub("_factset_.*[.]rds$", "", c(
+    factset_financial_data_filename,
+    factset_entity_info_filename,
+    factset_entity_financing_data_filename,
+    factset_fund_data_filename,
+    factset_isin_to_fund_table_filename,
+    factset_iss_emissions_data_filename
+  )))
+
```

Comment:
What happens if the `unique` returns a length more than 1? Suggest putting a check here (or somewhere) to at least warn if more than one timestamp is found (in case user doesn't check the exported `parameters`, below)

Reply:
See original PR description